Feature/192347 disable system generated msg removal for admin and moderator

sppanday opened the pull request:
**Administrator or Moderator of chatroom should not be able to see remove option and remove system generated messages that is available on panel.** 

This can be tested by hovering on system messages in chat room such as "person x has joined the room...", right click, and remove option wont be available for moderator and admin just for system messages. 

Admin and moderator will still have ability to delete chat messages which is default behavior and is required for other parent user story linked in this user story.

sppanday committed:
disabled remove button for admin/moderator for deleting system notification  messages

Rintan committed:
Translated using Weblate (Japanese)

Currently translated at 50.2% (1398 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/ja/

IhorHordiichuk committed:
Translated using Weblate (Ukrainian)

Currently translated at 53.6% (1492 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/uk/

Commit:
Translated using Weblate (Japanese)

Currently translated at 50.2% (1398 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/ja/

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (2780 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/it/

Commit:
Translated using Weblate (Sinhala)

Currently translated at 0.2% (7 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/si/

roeltm committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (2780 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/nl/

waclaw66 committed:
Translated using Weblate (Czech)

Currently translated at 100.0% (2780 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/cs/

MamasLT committed:
Translated using Weblate (Lithuanian)

Currently translated at 68.5% (1905 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/lt/

a22sc committed:
Translated using Weblate (German)

Currently translated at 99.3% (2761 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/de/

libexus committed:
Translated using Weblate (German)

Currently translated at 99.3% (2761 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/de/

thibaultamartin committed:
Translated using Weblate (French)

Currently translated at 100.0% (2780 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/fr/

turt2live committed:
Fixed review concerns

iapellaniz committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2780 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/es/

HelderFSFerreira committed:
add clipboard write permission

dbkr committed:
Merge pull request #5707 from matrix-org/dbkr/audiostream

Option for audio streaming

dbkr committed:
Merge pull request #5722 from matrix-org/dbkr/fix_widget_resize

Fix widget resizing

jryans committed:
Merge pull request #5725 from HelderFSFerreira/addClipboardWritePermission

Add clipboard write permission for widgets

anoadragon453 committed:
Fix some ineffective placeholders in a couple log lines (#5729)

Was producing `Presence: %s unavailable` in the logs.

turt2live committed:
Add a quick sender check to isLastSuccessful

t3chguy committed:
Show suggested rooms from the selected space

turt2live committed:
Merge pull request #5699 from matrix-org/travis/message-states

Improve message sending states to match new designs

turt2live committed:
Merge pull request #5719 from matrix-org/travis/file-representation

UI refresh for uploaded files

turt2live committed:
Merge pull request #5723 from matrix-org/travis/uploads

Refresh UI for file uploads

jryans committed:
Merge pull request #5624 from SimonBrandner/show-room-name

Display room name in pills instead of address

t3chguy committed:
Tidy up TemporaryTile now that it isn't temporary

dbkr committed:
Fix units of TURN server expiry time

panagiotis-c committed:
address PR comments

cleanup, change to isEncrypted, comments

a22sc committed:
Translated using Weblate (German)

Currently translated at 99.5% (2767 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/de/

libexus committed:
Translated using Weblate (German)

Currently translated at 99.6% (2769 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/de/

a22sc committed:
Translated using Weblate (German)

Currently translated at 99.6% (2769 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/de/

turt2live committed:
Fix types for replaceableComponent

This is to make it work in TS files

turt2live committed:
Make debugging skinning problems easier

turt2live committed:
Remove a bunch of useless 'use strict' definitions

dbkr committed:
Merge pull request #5730 from matrix-org/dbkr/fix_turn_expiry_units

Fix units of TURN server expiry time

jryans committed:
Merge pull request #5713 from panoschal/improve-view-source

Display decrypted and encrypted event source on the same dialog

jryans committed:
Merge pull request #5717 from 2580ayush2580/ayush258013

Fix clicking on the avatar for opening member info requires pixel-perfect accuracy

jryans committed:
Merge pull request #5695 from robintown/invite-command-reason

Support sending invite reasons with /invite command

thibaultamartin committed:
Translated using Weblate (French)

Currently translated at 100.0% (2780 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/fr/

MamasLT committed:
Translated using Weblate (Lithuanian)

Currently translated at 68.5% (1907 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/lt/

t3chguy committed:
Fix React warning

t3chguy committed:
Pass OOB data for suggested rooms

t3chguy committed:
Update suggested room on join

t3chguy committed:
Tweak call handler stuff to not explode the room list on unsupported servers

t3chguy committed:
Tweak styling of manage rooms in space ux

turt2live committed:
Merge pull request #5733 from matrix-org/travis/skinning/pt2-debug

[SK-2] Make debugging skinning problems easier

turt2live committed:
Merge pull request #5732 from matrix-org/travis/skinning/pt1-ts

[SK-1] Fix types for replaceableComponent

turt2live committed:
Merge pull request #5735 from matrix-org/travis/use-strict-no

Remove a bunch of useless 'use strict' definitions

turt2live committed:
Fix sent markers disappearing for edits/reactions

Fixes https://github.com/vector-im/element-web/issues/16651

turt2live committed:
Add tooltips to sent/sending receipts

Fixes https://github.com/vector-im/element-web/issues/16649

jrschremmer commented on src/components/views/context_menus/MessageContextMenu.js:
```diff
@@ -351,7 +351,7 @@ export default class MessageContextMenu extends React.Component {
         }
 
-        if (isSent && this.state.canRedact) {
+        if (isSent && this.state.canRedact && mxEvent.getType() === 'm.room.message') {
```
Please add a configuration parameter, allowRoomAdminsToRedactSystemMessages which defaults to true, and we override it to false in our web app config.json file.  Remember we want to push our changes to the matrix repo and have the default behaviour result in no functional change.

turt2live committed:
Document behaviour of showReadReceipts=false for sent receipts

Commit:
Translated using Weblate (Esperanto)

Currently translated at 99.1% (2755 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/eo/

jryans committed:
Merge pull request #5738 from matrix-org/travis/message-state-fixes/tooltips

Add tooltips to sent/sending receipts

t3chguy committed:
Merge pull request #5736 from matrix-org/t3chguy/spaces4.5

Spaces suggested rooms support

jryans committed:
Ignore to-device decryption in the room list store

This avoids meaningless warnings about "unknown" rooms.

jryans committed:
Merge pull request #5740 from matrix-org/jryans/ignore-roomless

Ignore to-device decryption in the room list store

tateisu committed:
Translated using Weblate (Japanese)

Currently translated at 79.1% (2199 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/ja/

jryans committed:
Merge pull request #5737 from matrix-org/travis/message-states-fixes/edits

Fix sent markers disappearing for edits/reactions

jryans committed:
Tweak sent marker code style

jryans committed:
Merge pull request #5741 from matrix-org/jryans/sent-marker-style

Tweak sent marker code style

turt2live committed:
Merge pull request #5739 from matrix-org/travis/message-state-fixes/hidden-rr

Document behaviour of showReadReceipts=false for sent receipts

jelv committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (2780 of 2780 strings)

Translation: Element Web/matrix-react-sdk
Translate-URL: https://translate.element.io/projects/element-web/matrix-react-sdk/nl/

Commit:
Merge branch 'origin/develop' into Weblate.

jryans committed:
Merge pull request #5743 from RiotTranslateBot/weblate-element-web-matrix-react-sdk

Translations update from Weblate

RiotRobot committed:
Upgrade matrix-js-sdk to 9.9.0-rc.1

RiotRobot committed:
Prepare changelog for v3.16.0-rc.1

RiotRobot committed:
v3.16.0-rc.1

RiotRobot committed:
Prepare changelog for v3.16.0-rc.2

RiotRobot committed:
v3.16.0-rc.2

turt2live committed:
Change read receipt drift to be non-fractional

I suspect this is what is causing issues in Firefox for read receipts not falling down.

SimonBrandner committed:
Use resourceId in formatted body

Signed-off-by: Šimon Brandner <simon.bra.ag@gmail.com>

t3chguy committed:
Properly gate SpaceRoomView behind labs

t3chguy committed:
Merge pull request #5750 from matrix-org/t3chguy/spaces4.6

[Release] Properly gate SpaceRoomView behind labs

turt2live committed:
Merge pull request #5746 from matrix-org/travis/fix-rr-rel

[Release] Change read receipt drift to be non-fractional

SimonBrandner committed:
Use resourceId in plain body

Signed-off-by: Šimon Brandner <simon.bra.ag@gmail.com>

RiotRobot committed:
Upgrade matrix-js-sdk to 9.9.0

RiotRobot committed:
Prepare changelog for v3.16.0

RiotRobot committed:
v3.16.0

sppanday committed:
Merge branch 'master' of https://github.com/matrix-org/matrix-react-sdk into fix/user-setting-merge-conflict

jrschremmer committed:
Merge pull request #10 from fhirfactory/fix/user-setting-merge-conflict

Fix/user setting merge conflict

sppanday committed:
changes for logo, hide popup, skip prelogin prompt

sppanday committed:
login prompt checks if e2e is enabled from server

sppanday committed:
secondary logo - increased gap

sppanday committed:
cleaned up code based on #12

jrschremmer committed:
Code review:
Rename ShowLiberateToCommunicateText to ShowLiberateYourCommunicationText;
Centralised the common logic in MatrixChat.tsx and createRoom.ts into WellKnownUtils.isE2EEEnabledInWellKnown;
Default LookForWellKnownConfigFromHomeServer to false, so the default element behaviour is maintained;

jrschremmer committed:
In WellKnownUtils.ts changed isE2EEEnabledInWellKnown to lookup the new enabled property and isE2EEDefaultedOnInWellKnown to lookup the existing default property

jrschremmer committed:
Merge pull request #12 from fhirfactory/feature/lingo-branding

Changes required for branding, hide of pop ups

sppanday committed:
feature documentation for rebranding (221259)

sppanday committed:
readjusted logo gap and positioning

sppanday committed:
spelling correction

sppanday committed:
hide password and homeserver name

sppanday committed:
config items for hiding item on splash screen

sppanday committed:
skip welcome screen to login screen

sppanday committed:
added appropriate img descr...for WCAG compliance

sppanday committed:
wording correction on well known util.

sppanday committed:
changed 'splashScreen' term to 'loginScreen'

sppanday committed:
made entire secondary logo configurable

sppanday committed:
Info update & variable change for login screen UI

sppanday committed:
cleanedup on branding description on feature doco

sppanday committed:
reordered secondary logo variable for readability

sppanday committed:
secondary logo config renamed

sppanday committed:
feature documentation updated

jrschremmer committed:
221259: Code Review - Added loginScreen.allowEmailBasedAuthentication;
Make loginScreen options explicit boolean config values and use the Nullish coalescing operator, to safely assign default boolean values to optional config values that provide boolean values;
Add link to Optional chaining operator;

jrschremmer committed:
Merge pull request #13 from fhirfactory/feature/221259-rebranding-changes

Feature/221259 rebranding changes

sppanday committed:
moved liberate your communicate outside setting

sppanday committed:
aligned logo and color as per requirement

Commit:
rebranding changes

Commit:
Merge branch 'feature/221259-rebranding-changes2' of https://github.com/fhirfactory/pegacorn-communicate-matrix-react-sdk into feature/221259-rebranding-changes

# Conflicts:
#	src/components/structures/HomePage.tsx

Commit:
Home page redesign

Commit:
Tabbed view act logo fix

Commit:
Login page rebranding UI changes

samridhis committed:
Updated string name Signin to Login on the login page

sppanday committed:
implemented role directory detail page view

sppanday committed:
translate roles with enums

sppanday committed:
config values for roles and service directory

sppanday committed:
directory view cleanup

sppanday committed:
aligned data and cleaned up role detail view

sppanday committed:
moved table logic separately for better separation

sppanday committed:
separated role detial api call

sppanday committed:
translated value field with enum

sppanday committed:
removed duplicate data, enum cleanup

Commit:
change was arleady part of aether host file so reverting it back from the shared source.

Commit:
updated the configuration for Log in instead of Sign in.

samridhis committed:
tabbed view changes display using condition in configration

samridhis committed:
Merge branch 'feature/222284-Lingo-Re-Branding-and-UI-Update' of https://github.com/fhirfactory/pegacorn-communicate-matrix-react-sdk into feature/222284-Lingo-Re-Branding-and-UI-Update

samridhis committed:
reverting back deletion of configuration changes.

sppanday committed:
method name change

samridhis committed:
code review comment to display act logo on login page at the auth header .

sppanday committed:
file name changed for role detail view

Commit:
replace act with secondary in regards to logo

Commit:
removed !imortant tag as it was not needed

Commit:
moved landing page logo height style into config file.

Commit:
reverted back removal of width.

sppanday committed:
implemented first,last name on avatar

sppanday committed:
role drop down on left panel

sppanday committed:
feature documentation for role directory

sppanday committed:
text cleanup

sppanday committed:
logic to display role directory dialog

sppanday committed:
logic to trigger role search dialog to apear

sppanday committed:
logic to show drop down for roles on left

sppanday committed:
removed unused function on role detail view

sppanday committed:
configs for role search

Commit:
latest configuration changes for branding UI changes discussed on 03rd may 2021

sppanday committed:
removing translation for role directory drop down

samridhis committed:
removed unused variables from config.ts file

samridhis committed:
tabbed view footer image url and alt text passing onto configuration file

samridhis committed:
revert back the color change.

samridhis committed:
revert back text-align changes

samridhis committed:
code review comments fix

sppanday committed:
removed bunch of console.log

samridhis committed:
$accent-color changed back to default and moved to aether-host-files

Commit:
pushing the changes

Commit:
removed unused variable

Commit:
Merge branch 'feature/222284-Lingo-Re-Branding-and-UI-Update' of https://github.com/fhirfactory/pegacorn-communicate-matrix-react-sdk into feature/222284-Lingo-Re-Branding-and-UI-Update

# Conflicts:
#	res/themes/light/css/_light.scss
#	src/config.ts

Commit:
externalise authheader alignment.

Commit:
removed newly added variable as its moved to common

Commit:
$authfooter-text-align:alighnment changes

samridhis committed:
disable_3pid_login fix

jrschremmer committed:
CODE REVIEW of 222284: Revert changes to files where no functional change remained (e.g. _common.scss, AuthHeaderLogo.js, LanguageSelector.js);
Revert change to PasswordLogin.tsx, as not required, as this code is hidden if the room server only supports SSO login, so we control this behaviour via that method instead;
In the themes move the definition of $authfooter-text-align:center to be grouped with the other $auth variables, which is then just above $dark-panel-bg-color in all themes;
In _light.scss move the new mx_TopBanner styles up from the bottom of the file to avoid merge conflicts with new lines added to element;

samridhis committed:
as per review comments

samridhis committed:
documentation update for newly added configuration changes.

sppanday committed:
Directory - role search functionality

sppanday committed:
config item for avatar color

sppanday committed:
made sorting to be alphabetically via config

sppanday committed:
feature documentation for role directory search

sppanday committed:
minor changes on Filter by favorite events

samridhis committed:
initial fixes

samridhis committed:
revert back wrong push meant for different story

sppanday committed:
readjusted role detail fetch function

sppanday committed:
enhanced documentation for role directory search

sppanday committed:
Finalised avatar color with configurable parameter

sppanday committed:
change 'invite' to 'discussion'

sppanday committed:
fixed bug on filled status not updating correctly

sppanday committed:
Added practitioners fulfilling role

sppanday committed:
indexing in view for role fulfilling

sppanday committed:
pointed role detial api to right api

sppanday committed:
added api prefix

sppanday committed:
re ordered render method, variables renamed

sppanday committed:
get rid of extra space

sppanday committed:
cleanup on role text

sppanday committed:
added role icon in homepage tile

sppanday committed:
added default to avatar colors not found in config

sppanday committed:
improved conditional checking for role search

sppanday committed:
Better way to find name from given matrix id

sppanday committed:
simplified avatar to have first/last name initials

sppanday committed:
refactored avatar name/id matching scenario

sppanday committed:
Changed text on left panel as we dont support
explore public room so text has been rewored

sppanday committed:
included null/undefined check

sppanday committed:
bunch of refactor on role directory for default

sppanday committed:
simplified directory search boolean check

jrschremmer committed:
CODE REVIEW: 222284-Lingo-Rebranding - Consolidate the different previous solution used for branding, in particular move the styling configuration out of config.json and into /res/themes/light/css/_light.scss, as some of the styling in config.json was being overwritten by /res/themes/light/css/_light.scss;
Remove config for secondary logos that was no longer required, to simplify testing. If require again the implementation can be referenced via the check in history;
Rollback changes to the following files as they are no longer required: src/i18n/strings/en_EN.json, src/i18n/strings/en_US.json, src/components/views/auth/PasswordLogin.tsx;
Fix the Login Screen impl, so it works for both use cases of the room server having password authentication enabled and not enabled;
Add the background gradient to the Login Screen;
Fix the implementation in _TabbedView.scss so the bottom div for the secondary logo, didn't impede on the right hand panel;
Change config starting with display* to show*, to be consistent with previous config names;

jrschremmer committed:
Fix whitespace

jrschremmer committed:
Re-added loginScreen.showHomeServerInfo;

jrschremmer committed:
Merge pull request #15 from fhirfactory/feature/222284-Lingo-Re-Branding-and-UI-Update

Feature/222284 lingo re branding and UI update

jrschremmer committed:
Re-add the code that updates the "Sign in" to "Log in" text if password authentication is enabled on the home server

jrschremmer committed:
Merge pull request #19 from fhirfactory/feature/222284-Lingo-Re-Branding-and-UI-Update

feature/222284-Lingo-Re-Brand

jrschremmer committed:
Merge branch 'master' of https://github.com/fhirfactory/pegacorn-communicate-matrix-react-sdk into feature/search-user-in-directory

# Conflicts:
#	feature-communicate-list.md
#	src/components/structures/HomePage.tsx
#	src/config.ts

sppanday committed:
Refactored avatar logic , added description

sppanday committed:
initialized values in constructor to false,
removed redundant true/false

sppanday committed:
added new config for left panel default text
caption change to the text that client wants through config

sppanday committed:
added new text for landline extension

sppanday committed:
centralised shared functions to utility folder

sppanday committed:
View renamed to adjust with service, ppl directory

sppanday committed:
renamed roleId to general queryId

sppanday committed:
Added error for directory search to display in UI

sppanday committed:
removed randomly selected color selection for
better user experience

sppanday committed:
Using 'view' instead of 'view detail'

sppanday committed:
removed defaultToOtherColor, randomlySelectedColor

sppanday committed:
aligned feature doc description to match with flag

sppanday committed:
tidied up space, text etc.

sppanday committed:
added else condition on avatar color selection

sppanday committed:
changed javascript to nicer format/plain english.

sppanday committed:
refactored terms, realignes import references

sppanday committed:
replaced redundant else if with else

sppanday committed:
Capitalized Directory Search terminologies

sppanday committed:
moving categoryId check to new line

sppanday committed:
handled error when calling favorites api

sppanday committed:
handled error, simplified boolean

sppanday committed:
removed instanceOf/ not needed in catch block

sppanday committed:
Moving code to newline to where it is required

sppanday committed:
refactored, centralised how avatar is named

sppanday committed:
simplified error to textual format in catch block

sppanday committed:
made search results unique

sppanday committed:
changed contact to office phone, mobile phone type

sppanday committed:
phone type refactor, removed simplifiedID from UI

sppanday committed:
aligned new server search results for roles

sppanday committed:
removed extra dash on contact

sppanday committed:
minor change in contact displayed for directory

sppanday committed:
removed redundant data injections on Member

sppanday committed:
removed redundant  fields in Member object

sppanday committed:
cleaned up phone number in enum

sppanday committed:
added missing documentation for ke formatting

samridhis committed:
integration of role selector redirect

sppanday committed:
reverted change as it showed some problem

samridhis committed:
validated the fix and added console.log to be able to understand the behaviour on test

samridhis committed:
fetch variables from config file

samridhis committed:
role selector redirect first time using config file values

samridhis committed:
fixing empty condition for ref

sppanday committed:
role directory button and translation

sppanday committed:
minor tweak in favorite

sppanday committed:
added pager, fax, workphone text labels

jrschremmer committed:
CODE REVIEW: revert all changes to src/PageTypes.js, as no changes are required;
In Avatar.ts remove redundant code;
In HomePage.tsx, move the directory tiles above the start a chat tile to encourage users to communicate to role, as agreed in the App Playback;
In InviteDialog.tsx and DirectoryDetailView.tsx, centralise error logging to the console, and re-use an existing internationalised phrase for the error message shown to the user, instead of introducing a new hard coded English only phrase;
In InviteDialog.tsx, fix another scenario where serverResultsMixin can be set to undefined and comment out "Showing X records of Y" until it is working;
Rename config.use_custom_text_label_on_left_menu to config.left_hand_nav_help_text for clarity
In RoomList.tsx, move the config.left_hand_nav_help_text above the default conditional code for showing text in the same location, to make it obvious that config.left_hand_nav_help_text overrides any default behaviour; show roles based on config.showRoleDirectory;
In config.ts, rename use_custom_text_label_on_left_menu to left_hand_nav_help_text, for consistency and clarity;
In formatEmailUtil.ts, formatKeyValueUtil.ts, formatPhoneNumberUtil.ts and strings.ts, centralise code;
In LeftPanel.tsx, only show the Explore Public Rooms button if config.showExplorePublicRooms is true;
In directory-enums.ts, match the label to those used in the IAM application;

sppanday committed:
added regex back as role dir..avatar did not work

sppanday committed:
rename api_base_path to communicate_api_base_path

jrschremmer committed:
CODE REVIEW: In Avatar, remove the repeated regex and instead handle when getInitialLetter(lastName) is undefined;
Create DirectoryService.ts and move all directory service api calls to this class, to separate concerns, centralise api calling code and make it easier in the future to introduce mock service implementations;
Only call the search practitioner search once (before it was calling it twice: once to get the totalRecordCount and a second call to get the search results);

jrschremmer committed:
Merge pull request #17 from fhirfactory/feature/search-user-in-directory

Feature/search user in directory

samridhis committed:
as per review comments removed extra code

Commit:
as per code review comments

jrschremmer committed:
Merge pull request #21 from fhirfactory/feature/225816-Display-Selected-Roles-In-Lingo-Web-App

Feature/225816 display selected roles in lingo web app

sppanday committed:
Merge branch 'feature/192347-disable-msg-removal' of https://github.com/fhirfactory/pegacorn-communicate-matrix-react-sdk into feature/192347-disable-msg-removal
